.github/workflows/main.yml

```diff
@@ -1,21 +1,85 @@
-name: CI on Windows
+name: CI
 
 on: [push, pull_request]
 
 jobs:
-  build:
+  ubuntu:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["py35", "py36", "py37", "py38", "nightly"]
+
+        include:
+          - name: "py35"
+            python: "3.5"
+            tox_env: "du12"
+
+          - name: "py36"
+            python: "3.6"
+            tox_env: "du13"
+
+          - name: "py37"
+            python: "3.7"
+            tox_env: "du14"
+
+          - name: "py38"
+            python: "3.8"
+            tox_env: "du15"
+            use_coverage: true
+
+          - name: "nightly"
+            python: "3.10-dev"
+            tox_env: "du16"
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python }}
+        uses: actions/setup-python@v2
+        if: matrix.python != '3.10-dev'
+        with:
+          python-version: ${{ matrix.python }}
+
```

Comment:
We have to choose the style of YAML; sparse or dense. The old YAML (for windows) contains a few blank lines. But this PR adds many blank lines to make YAML sparse (maybe for readability). To keep the file consistent, it is not good to mix styles on the single file. Please let me know your opinion. Personally, I prefer a dense style a bit.

Reply:
I don't have any preference here. Whatever works for you is fine for me. I'll change the style to dense.